student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,9 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    
+    const char *my_user = my_username();
+    const char *conf_user = malloc_username_from_conf_file();
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(conf_user,my_user,"USER NAMES DO NOT MATCH");
```

Comment:
@ceca5556 what happens to the string allocated by `malloc_username_from_conf_file` here?